docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -469,7 +469,8 @@ testImplementation("io.rest-assured:rest-assured")
 testImplementation("io.quarkus:quarkus-junit5")
 ----
 
-[[integration-testing-wiremock]]
+ifndef::no-deprecated-test-resource[]
```

Comment:
This option is used for Wiremock testing for applications which do not use Keycloak